parallel-install/pkg/deployment/overrideinterceptors.go

```diff
@@ -260,3 +260,25 @@ func (i *InstallLegacyCRDsInterceptor) Undefined(overrides map[string]interface{
 func NewInstallLegacyCRDsInterceptor() *InstallLegacyCRDsInterceptor {
 	return &InstallLegacyCRDsInterceptor{}
 }
+
+// This struct is introduced to ensure kcproxy gets disabled on kiali and tracing component
+// It can be removed when kcproxy si removed
+type DisableKCProxyInterceptor struct{}
+
+func (i *DisableKCProxyInterceptor) String(value interface{}, key string) string {
+	return fmt.Sprintf("%v", value)
```

Comment:
Shouldn't this always return false? Otherwise a printout of the overrides could say `kiali.kcproxy.enabled: true` although in reality it got disabled.

Reply:
Oh yes, you are right! I will change